.github/workflows/CI.yml

```diff
@@ -274,3 +274,40 @@ jobs:
         run: |
           SET CHERE_INVOKING=1
           C:\cygwin64\bin\bash -l -c "bash dev/ci.sh"
+
+  slack-notification-on-failure:
+    name: Send slack notification on CI failure
+    needs:
+      - test-unix
+      - test-cygwin
+    if: ${{ always() && github.event_name == 'push' && github.repository == 'gap-system/gap' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get branch name
+        id: get-branch
```

Comment:
Seems this is unused? You access the output of this step via `steps.vars.outputs.branch` below (I did not check whether that works, though)

Reply:
Yes, that must have been broken. I'm updating this. I didn't realise because I only tested my MWE version of this action. I'm doing a proper test on my fork now.

Reply:
The four latest runs on https://github.com/ssiccha/gap/actions should now trigger all cases of this new job.

Reply:
Both notifications were sent to #status-action-tests.